Formula/vim.rb

```diff
@@ -19,6 +17,12 @@ class Vim < Formula
   LANGUAGES_OPTIONAL = %w[lua mzscheme python3 tcl]
   LANGUAGES_DEFAULT  = %w[perl python ruby]
 
+  if MacOS.version >= :mavericks
```

Comment:
Heh, right you are. I may just give up reviewing PRs, it's not going well lately 🙈.